packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/template/XmlSequenceWrapperTemplate.java

```diff
@@ -41,6 +41,9 @@ public final void write(XmlSequenceWrapper xmlSequenceWrapper, JavaFile javaFile
 
         Set<String> imports = xmlSequenceWrapper.getImports();
 
+        Annotation.METADATA.addImportsTo(imports);
+        Annotation.METADATA_PROPERTIES.addImportsTo(imports);
```

Comment:
Curious why the generated code does not change? I think we've added imports. Do I misunderstand something?
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4936208&view=logs&j=81a719ea-44f7-5728-14cb-b369713e6844&t=0eba2929-bc33-5548-9864-6177e6ffc00e